lib/datadog/tracing/configuration/ext.rb

```diff
@@ -10,6 +10,7 @@ module Ext
         # @public_api
         module SpanAttributeSchema
           ENV_SPAN_ATTRIBUTE_SCHEMA = 'DD_TRACE_SPAN_ATTRIBUTE_SCHEMA'.freeze
+          ENV_SERVICE_NAME_OVERRIDE = 'DD_TRACE_REMOVE_CLIENT_SERVICE_NAMES_ENABLED'.freeze
```

Comment:
sorry for the last minute change, but it was recently changed to `DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED`

Reply:
Done